### PR TITLE
Add reference to paper

### PR DESCRIPTION
**Pull request by frankenjoe**
Adds a reference to associated paper.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,11 +65,20 @@ valence.
...
 
 ## Citation
 
-TBA
+If you use our model in your own work, please cite the following paper:
+
+```bibtex
+@article{wagner2022dawn,
+  title={Dawn of the transformer era in speech emotion recognition: closing the valence gap},
+  author={Wagner, Johannes and Triantafyllopoulos, Andreas and Wierstorf, Hagen and Schmitt, Maximilian and Eyben, Florian and Schuller, Bj{\"o}rn},
```
Comment (hagenw) [inline README.md]:
```suggestion
  author={Wagner, Johannes and Triantafyllopoulos, Andreas and Wierstorf, Hagen and Schmitt, Maximilian and Eyben, Florian and Schuller, Bj{\"o}rn W., and Burkhardt, Felix},
```

Comment (hagenw) [reply README.md]: Besides adding Felix, I also added "W." to Björns name as he used it in the paper.

File: README.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ If you use our model in your own work, please cite the following paper:
 ```bibtex
 @article{wagner2022dawn,
   title={Dawn of the transformer era in speech emotion recognition: closing the valence gap},
-  author={Wagner, Johannes and Triantafyllopoulos, Andreas and Wierstorf, Hagen and Schmitt, Maximilian and Eyben, Florian and Schuller, Bj{\"o}rn and Burkhart, Felix},
+  author={Wagner, Johannes and Triantafyllopoulos, Andreas and Wierstorf, Hagen and Schmitt, Maximilian and Burkhardt, Felix and Eyben, Florian and Schuller, Bj{\"o}rn W.},
```
Comment (hagenw) [inline README.md:80]: This is not the order currently shown in the paper.

Comment (frankenjoe) [reply README.md:80]: I re-submitted a new version of the paper on arxiv, which changes the author list to:

image

I changed it accordingly (including the missing W.)

Comment (frankenjoe) [reply README.md:80]: New version us up: https://arxiv.org/abs/2203.07378

Comment (hagenw) [reply README.md:80]: Thanks, updated version is online now.

**Pull request merged**